### PR TITLE
Adding aws client image

### PR DESCRIPTION
**Pull request by etsauer**
#### What does this PR do?

Adds a docker image with tooling for AWS provisioning, including the ec2 client, boto libraries, and ansible. Everything needed to run `openshift-ansible/bin/cluster aws` commands
#### How should this be manually tested?

Clone openshift-ansible, setup an ec2 environment and follow instructions for AWS provisioning using bin/cluster
#### Is there a relevant Issue open for this?

n/a
#### Who would you like to review this?

/cc @sabre1041 @oybed @JaredBurck

### CODE REVIEW

File: docker/aws-client/run.sh [hunk 1]
```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+# Run.sh - Script to build and run a Docker container to facilitate communicate with OpenStack
```
Comment (JaredBurck) [inline docker/aws-client/run.sh]: Should say AWS

File: docker/aws-client/run.sh [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+fi
+
+if [ -d $SSH_DIR ]; then
+	SSH_VOLUME="-v ${SSH_DIR}:/mnt/.ssh:z"
+else
+	echo "Warning: SSH Directory not found"
+fi
+
+
+echo "Starting OpenStack Client Container...."
```
Comment (JaredBurck) [inline docker/aws-client/run.sh]: Should say AWS

File: docker/aws-client/run.sh [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+REPOSITORY_VOLUME=""
+
+
+usage() {
+    echo "
+     Usage: $0 [options]
+     Options:
+     --credsfile=<file>             : File containing AWS API credentials (Default: ~/.aws_creds/)
+     --configfile=<file>            : File containing ec2 environment variables (Default: ~/.aws_env)
+     --image-name=<name>           : Name of the image to build or use (Default: rhtconsulting/rhc-openstack-client)
```
Comment (JaredBurck) [inline docker/aws-client/run.sh]: Should say (Default: rhtconsulting/aws-client-tools)

**Comment by etsauer** [comment]
@JaredBurck fixed the typos. Anything else?

**Comment by sabre1041** [comment]
@JaredBurck are we good to merge?

**Comment by JaredBurck** [comment]
@sabre1041 working on final verifications.  should be good shortly.

**Comment by etsauer** [comment]
cc: @mdanter

**Pull request closed without merging**